jquery.daterangepicker.js

```diff
@@ -1259,6 +1260,15 @@
 			{
 				$(self).data('date-picker-opened',false);
 			});
+			if (opt.end && !(justClose === true)) {

```

Comment:
I'm getting errors now because `justClose` is not defined prior to here, or anywhere in this file.